src/aks-preview/azext_aks_preview/_validators.py

```diff
@@ -372,3 +372,13 @@ def validate_label(label):
                        "characters, '-', '_' or '.', and must start and end with an alphanumeric character" % label)
 
     return {kv[0]: kv[1]}
+
+
+def validate_nodepool_mode(namespace):
+    """Validates the nodepool mode string."""
+    if namespace.mode is not None:
+        if namespace.mode == '':
+            return
```

Comment:
can we combine these three lines to `if not namespace.mode` ?

Reply:
Removed this validation because the parameter type is set as enum type: get_enum_type([CONST_NODEPOOL_MODE_SYSTEM, CONST_NODEPOOL_MODE_USER])
Enum type can convert and validate the input. So both "System" and "system" will be converted to "System".